### PR TITLE
Feat/geolocation

### PR DESCRIPTION
**Pull request by AndrewHyeon-JahnelGroup**

### CODE REVIEW

**Review by picodedan — CHANGES_REQUESTED** [review]
in server.js you appear to have added a redirect before the DB call and return,  not sure why we would need to be redirected to index.js there, but that pathing doesn't work. 
It appears you may have removed the gitignore file?

you'll need to fix the above and then do a pull and rebase from the master image to pull in the changes that have already been merged.  your code appears to be missing the client side changes that were merged yesterday.

File: server/server.js [hunk 1]
```diff
@@ -10,8 +10,9 @@ app.use('/', express.static(path.join(__dirname, '../')));
 app.use('/', express.static(path.join(__dirname, '../client')));
 
 //routes
-app.get('/images', (req, res, next) => {
-  console.log('GET REQUEST IN SERVER', req.body)
+app.get('/images', function(req, res, next) {
+  console.log('GET REQUEST IN SERVER')
+  res.redirect('/index.js');
```
Comment (picodedan) [inline server/server.js:15]: why are we redirecting here?

File: .gitignore [hunk 2]
```diff
@@ -1,29 +0,0 @@
-### node etc ###
-
```
Comment (picodedan) [inline .gitignore:2]: why is gitignore being removed?

**Comment by AndrewHyeon-JahnelGroup** [comment]
fixing

**Pull request closed without merging**